packages/amplication-client/src/Application/ApplicationAuthSettingForm.tsx

```diff
@@ -105,19 +105,19 @@ function ApplicationAuthSettingForm({ match }: Props) {
                 </p>
```

Comment:
use this text instead:

Select the authentication method to be used in the generated app.